src/components/Modals/Send/hooks/useSendDetails/useSendDetails.tsx

```diff
@@ -50,30 +50,30 @@ export const useSendDetails = (): UseSendDetailsReturnType => {
   const accountId = useWatch<SendInput, SendFormFields.AccountId>({
```

Comment:
If we use object destructuring at the top and we can tighten this file right up:

```ts
const { assetId } = asset
```

Then all of the:

```ts
selectPortfolioFiatBalanceByFilter(state, { assetId: asset.assetId, accountId }),
```

Become:

```ts
selectPortfolioCryptoHumanBalanceByFilter(state, { assetId, accountId }),
```